src/Slices/Notification/UI/Drawer/Item.tsx

```diff
@@ -78,7 +70,7 @@ const ActionList: React.FC<Props> = ({ notification, onUpdate }) => {
           onClick={() => onUpdate({ read: false })}
           isDisabled={!notification.read}
         >
-          {words("notification.drawer.unread")}
+          {words("notification.unread")}
         </DropdownItem>,
         <DropdownItem
```

Comment:
The `Show all notifications` item doesn't look like it's a navigation item. Maybe we should style it as a link?

Reply:
The patternfly dropdown component mentions having links in the dropdown, and they are not styled differently.
I also do not see anything in the guidelines about styling them like a link.
So I think it's best to just leave this as is.